ote/src/clj/ote/services/admin.sql

```diff
@@ -16,3 +16,35 @@ SELECT o.id, o."name" as "operator", o."business-id" as "business-id",
 -- name: delete-transport-operator
 -- Delete all operator data except published external interface data from ckan
 SELECT del_operator(:operator-group-name);
+
+-- name: search-interfaces-by-operator
+-- Find published interfaces using operator name
+SELECT i.id as "interface-id", ts.id as "service-id",
+op.id as "operator-id", op.name as "operator-name", op.email as "operator-email", op.phone as "operator-phone", op.gsm as "operator-gsm",
+ts.name as "service-name", ts."contact-phone" as "service-phone", ts."contact-email" as "service-email",
+i."data-content" as "data-content", (i."external-interface").url as url,
+i.format as format, i."gtfs-imported" as imported, i."gtfs-import-error" as "import-error", i."gtfs-db-error" as "db-error"
+  FROM "transport-operator" as op, "transport-service" as ts, "external-interface-description" as i
+ WHERE op.name ilike :name
+   AND ts."published?" = TRUE
+   AND ts."transport-operator-id" = op.id
+   AND i."transport-service-id" = ts.id
+   AND ('GTFS' = ANY(i.format) OR 'Kalkati.net' = ANY(i.format))
+ GROUP BY ts.id, op.id, i.id
+ ORDER BY i.format ASC, i."gtfs-import-error" DESC;
```

Comment:
Check both parameters in one query, by `(:someparam::INTEGER IS NULL OR my.column = :someparam)`